tools/clang_tidy/pubspec.yaml

```diff
@@ -25,8 +25,8 @@ dependencies:
 dev_dependencies:
   async_helper: any
   expect: any
-  litetest: any
   smith: any
+  test: any
```

Comment:
We can't add a dependence on `package:test` to Engine Dart language tests due to the difficulty it introduces in fetching dependencies for the repo. Please retain using `litetest` for `clang_tidy`. Some context is in https://github.com/flutter/flutter/issues/82134.

Reply:
Looks like the dart tool isn't modular such that it can use `litetest` with `dart test test`.  If we can't make 'dart test test' work this PR is pointless.  We are stuck with a bespoke testing setup.  Ditching the PR.